packages/allow-scripts/test/prepare.js

```diff
@@ -0,0 +1,20 @@
+const { execSync } = require('child_process')
+const { readdirSync } = require('node:fs')
+const { join, normalize } = require('node:path')
+
+// Execute allow-scripts command inside each test project directory
+readdirSync(join(__dirname, '../test/projects/'), { withFileTypes: true })
+  .filter((p) => p.isDirectory())
+  .forEach((dir) => {
+    try {
+      execSync(`node ${normalize('../../../src/cli.js')} auto --experimental-bins`, {
```

Comment:
I was eyeballing this and wondered why it was working, but it's because this uses `exec` and not `execFile`.  If it were `execFile`, you'd want to use `process.execPath` instead of `node`.

Reply:
Does it make sense to leave a comment here with this information?